### PR TITLE
Add test setup instructions

### PR DESCRIPTION
**Pull request by stevemessick**
Part of this was copied from the Dart plugin readme.
@devoncarew @pq

### CODE REVIEW

**Comment by pq** [comment]
This looks awesome.  Is there any way we can commit this run config so it just shows up without any need to manually create it?

**Comment by stevemessick** [comment]
If you can get it added it to the intellij-community repository there
should be no problem. You'll have to submit a PR to Alex.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,3 +34,16 @@ file in the Dart plugin.
   - Choose Edit Configuration in the Run Configuration drop-down list.
   - Expand Defaults and verify that Flutter is present.
   - Click [+] and verify that Flutter is present.
+
+In order to run unit tests you need to create a run configuration. The easiest way is to copy the
```
Comment (devoncarew) [inline CONTRIBUTING.md]: Perhaps add a new section: `## Running plugin tests`?

**Comment by devoncarew** [comment]
lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (stevemessick) [inline CONTRIBUTING.md]: That's not a bad idea. I think the whole thing could use some editing.

**Pull request merged**